docs/src/views/ColorPalette.tsx

```diff
@@ -0,0 +1,113 @@
+        ColorBox: colorBoxVariables,
+      },
+    }}
+  >
+    <ProviderConsumer
+      render={({ siteVariables: { colors, contextualColors, emphasisColors, naturalColors } }) => (
+        <DocPage title="Color Palette">
+          <Header as="h2">Introduction</Header>
+          <p>
+            The color palette for a theme has many requirements and constraints. There is a need to
+            be intentional and functional with color use. We analyzed existing frameworks and picked
+            the best ideas from them.
+          </p>
+          <p>
+            Each theme should match our color palette types fully. This will allow you to use our
+            theming features completely and keep your palette structured.
+          </p>
+
+          <Header as="h2">Primitive colors</Header>
+          <p>
+            This part of the palette includes only <i>black</i> and <i>white</i> colors, we decided
+            to separate by semantical ideas. There is nothing blacker than black and nothing whiter
+            than white.
+          </p>
+
+          <Grid columns={2}>
+            {_.map(['black', 'white'], color => (
+              <Grid.Column key={color}>
+                <ColorBox name={color} rounded size="big" value={colors[color]} />
+              </Grid.Column>
+            ))}
+          </Grid>
+
+          <Header as="h2">Natural colors</Header>
+          <p>
+            This part of palette includes nine colors that are the most frequently used among
+            popular frameworks. Each color includes ten gradients, this allows us to satisfy most
+            common needs. This decision is experienced from Material UI and allows to define more
+            variants than semantical naming (lightest, lighter, etc.).
+          </p>
+
+          <Grid columns={2}>
+            {_.map(naturalColors, (variants, color) => (
+              <Grid.Column key={color}>
+                <ColorBox name={color} rounded value={colors[color][500]} />
+              </Grid.Column>
+            ))}
+          </Grid>
+
+          <Header as="h2">Contextual colors</Header>
+          <p>
+            This part of the palette includes primary and secondary (i.e. brand) colors and
+            contextual colors. Contextual colors can be just aliases for natural colors.
+          </p>
+
+          <Grid columns={2}>
+            {_.map({ ...emphasisColors, ...contextualColors }, (variants, color) => (
+              <Grid.Column key={color}>
+                <ColorBox name={color} rounded size="big" value={colors[color][500]} />
+              </Grid.Column>
+            ))}
+          </Grid>
+
+          <Header as="h2">Text colors</Header>
+          <p>
+            Text variants are also separated as state color because in the most cases it's not
+            correct to use grey color for text.
+          </p>
+
+          <Segment>
+            {_.map(colors.text, color => (
+              <p key={color} style={{ color }}>
```

Comment:
note, first paragraph is almost unreadable because of the low visual contrast. As a client I would rather not expect to see this example on the 'guide' page where examples of suggested practices are provided. Just thinking if we could present it better somehow

![image](https://user-images.githubusercontent.com/9024564/49378831-e39aab00-f70d-11e8-8b2e-793e248ec45d.png)

Reply:
I reworked Text block, now it looks much readable:

![image](https://user-images.githubusercontent.com/14183168/49380070-6c6b1400-f719-11e8-9a08-d55065f5bb43.png)